syntax="proto3";

service Backend {
    rpc load_image (img_path) returns (image);
    rpc predict (img_predic_ruta) returns (prediction_result);
}

message image {
    string img_content = 1;
    int32 width = 2;
    int32 height = 3;
}

message img_path {
    string path = 1;
}

message prediction_result {
    string label_prediction = 1;
}

message img_predic_ruta {
    string path_img_predic_ruta = 1;
}
